docs/src/main/asciidoc/datasource.adoc

```diff
@@ -219,18 +219,21 @@ quarkus.datasource.url=jdbc:h2:tcp://localhost/mem:default
 quarkus.datasource.username=username-default
 quarkus.datasource.min-size=3
 quarkus.datasource.max-size=13
+quarkus.datasource.enable-metrics=true
```

Comment:
This is the `enable-metrics` property that enables metric collection, not exposition, and it is per data source. It is already in Quarkus, I'm only adding it to the documentation example, because you need to enable metrics collection if you also enable metrics exposition and don't want to have all values equal to zero.